controller/controllers/httpscertissuer_controller.go

```diff
@@ -55,6 +55,17 @@ type HttpsCertIssuerReconciler struct {
 func (r *HttpsCertIssuerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 
+	certMgrNs := corev1.Namespace{}
+	err := r.Get(ctx, client.ObjectKey{Name: CertManagerNamespace}, &certMgrNs)
```

Comment:
Should we watch the CertManagerNamespace?

Reply:
updated